libpod/info_linux.go

```diff
@@ -55,12 +49,6 @@ func (r *Runtime) setPlatformHostInfo(info *define.HostInfo) error {
 	}
 	info.Slirp4NetNS = define.SlirpInfo{}
 
-	cgroupVersion := "v1"
-	if unified {
-		cgroupVersion = "v2"
-	}
-	info.CgroupsVersion = cgroupVersion
```

Comment:
This is now completely gone?

Reply:
I had planned to set it earlier to v2, but then @nalind mentioned https://github.com/containers/buildah/pull/6424#discussion_r2430247292 . Hence the same change here. I can revert.

Reply:
I wonder if we have some folks using this field in scripts and such, I think printing v2 should be safe since we never get here as we have the early check in main/persistentPreRunE.

Reply:
done, set to v2.